src/downloadmanager.cpp

```diff
@@ -1321,6 +1321,21 @@ QString DownloadManager::getFileName(int index) const
   return m_ActiveDownloads.at(index)->m_FileName;
 }
 
+int DownloadManager::getDownloadIndex(QString filename) const
```

Comment:
Feels like this should not be in this PR?

Reply:
I think I added these because I thought I needed them to get the category ID from the download during the install but I was able to pull it with the post-install nexus API call instead. So this function is not needed.